Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift

```diff
@@ -123,34 +119,6 @@ struct ContentView: View {
                     }
                 }
             }
-
-            Button("Alipay") {
-                alipayPlugin.start(LogtoSocialPluginConfiguration(
-                    redirectTo: URL(string: "alipay://?app_id=some_app_id&state=foo")!,
-                    callbackUri: URL(string: "https://logto.dev/callback/alipay")!,
-                    completion: { url in
-                        print(url)
-                    },
-                    errorHandler: { error in
-                        print(error)
-                    }
-                ))
-            }
-
-            Button("Wechat") {
-                wechatPlugin.start(LogtoSocialPluginConfiguration(
-                    redirectTo: URL(
-                        string: "wechat://?app_id=wx89d03327bc26d757&state=foo&universal_link=https://logto.io/app/"
-                    )!,
-                    callbackUri: URL(string: "https://logto.dev/callback/wechat")!,
-                    completion: { url in
-                        print(url)
-                    },
-                    errorHandler: { error in
-                        print(error)
-                    }
-                ))
-            }
```

Comment:
This might be useful for some people, though. If we remove them, are we considering creating another demo for social sign-ins?

Reply:
yes, it should be a separate demo if needed

Reply:
LOG-2219